src/floating-focus.js

```diff
@@ -150,17 +150,30 @@ export default class FloatingFocus {
 		this.target.classList.remove('focus');
 	}
 
+	addPixels(pixels1, pixels2) {
+		const result = parseFloat(pixels1) + parseFloat(pixels2);
+		return !isNaN(result) ? `${result}px` : null;
+	}
```

Comment:
Or maybe better to make into a static method / even a util method outside of the class?
(The reason I did it like this is that e.g. repositionElement() also does nothing with this and yet still is a class method)

Reply:
Ah je hebt gelijk; eigenlijk kan de browser natuurlijk prima met `null` omgaan (als in het wordt alsnog naar `0px` geparsed) maar misschien toch wel mooier om dat expliciet te returnen...